specification/compute/resource-manager/Microsoft.Compute/preview/2020-09-30/sharedGallery.json

```diff
@@ -0,0 +1,1040 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "SharedGalleryServiceClient",
+    "description": "Shared Gallery Service Client.",
+    "version": "2020-09-30"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/sharedGalleries": {
+      "get": {
+        "tags": [
+          "SharedGalleries"
+        ],
+        "operationId": "SharedGalleries_List",
+        "description": "List shared galleries by subscription id or tenant id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationNameParamter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "sharedTo",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The query parameter to decide what shared galleries to fetch when doing listing operations.",
+            "enum": [
+              "tenant"
```

Comment:
does the user have to explicitly ask for tenant shared galleries or are they included by default in the GET without a "sharedTo" param?

Reply:
Current design for preview is that we return galleries shared to subscription by default. If user specify '?sharedTo=tenant', then we turn to list galleries shared the tenant. So yes, user have to explicitly ask for tenant shared galleries at this moment.

In the future, we might change the query parameter options to '?sharedTo=tenant|xxx|any', but for now, that's our design for preview.